workload/src/Samsung.Tizen.Build.Tasks/CheckApiVersion.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+using System.Text.RegularExpressions;
+using System.Xml.Linq;
+
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+using Samsung.Tizen;
+
+namespace Samsung.Tizen.Build.Tasks
+{
+    public class CheckApiVersion : Task
+    {
+        [Required]
+        public string ManifestApiVersion { get; set; }
+
+        [Required]
+        public string TargetFrameWork { get; set; }
```

Comment:
IMO, Using `TargetFrameworkIdentifier` and `TargetPlatformIdentifier` is more clear.  Please check these properties.

Reply:
`TargetFrameworkIdentifier` is more clear. I will fix.